utils/zerovec/src/zerovec/mod.rs

```diff
@@ -431,10 +491,7 @@ where
     /// zerovec.make_mut().push(12_u16.as_unaligned());
     /// assert!(matches!(zerovec, ZeroVec::Owned(_)));
     /// ```
-    //
-    // This function is crate-public for now since we don't yet want to stabilize
-    // the internal implementation details
-    pub(crate) fn make_mut(&mut self) -> &mut Vec<T::ULE> {
+    pub fn make_mut(&mut self) -> &mut Vec<T::ULE> {
```

Comment:
Should we call it `.to_mut()` to align with alloc::borrow::Cow::to_mut ?

Reply:
It's `make_mut()` on Arc and on VZV. But I do think to_mut makes a lil more sense

Reply:
Oh interesting.  `make_mut()` on Arc is a newer API.  So you changed my mind and now I think we should stick with `make_mut()`.